addons/pos_restaurant/static/src/app/product_screen/order_summary/order_summary.js

```diff
@@ -34,16 +36,24 @@ patch(OrderSummary.prototype, {
         this.pos.showScreen("FloorScreen");
     },
     showUnbookButton() {
+        if (this.pos.selectedTable) {
```

Comment:
we are adding a "Cancel order" button, so there's no need for this diff


https://github.com/odoo/odoo/pull/176129

Reply:
This is in master so for 17.4 I will keep this and remove it in master